### PR TITLE
Adjustment of API request and fix of bugs when using real backend

### PR DESCRIPTION
**Pull request by isabellakutger**
- Fixed header in post request and URL params of get request of new resources
- Fixed filter selection in stats component and put it in the right order
- Text filter now filtering while typing and resets when navigating
- Fixed pagination: 
     - offset resets when navigating to another table view
     - adjust the calculation of pages
     - immediate load when changing pages size or page
- Fixed API requests of bulk actions (start task, metadata update)
- Added remaining notifications for adding and deleting resources

### CODE REVIEW

File: modules/admin-ui-picard/app/src/components/events/partials/wizards/RenderField.js [hunk 1]
```diff
@@ -189,9 +189,15 @@ const EditableSingleSelect = ({ field, metadataField, text, editMode, setEditMod
                                 <option key={key} value={item.value}>{t(item.name)}</option>
                         ))
                     ) : (
-                        metadataField.collection.map((item, key) => (
+                        metadataField.id === "isPartOf" ? (
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/components/events/partials/wizards/RenderField.js:193]: write a comment for this check

File: modules/admin-ui-picard/app/src/selectors/recordingSelectors.js [hunk 2]
```diff
@@ -3,3 +3,4 @@
  */
 
 export const getRecordings = state => state.recordings.results;
+export const getTotalRecordings = state => state.recordings.results;
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/selectors/recordingSelectors.js]: Is this right? All other added `getTotal[Something]` used `state.[something].total`, not `state.[something].results` as it is here

File: modules/admin-ui-picard/app/src/thunks/tableThunks.js [hunk 3]
```diff
@@ -44,308 +44,422 @@ import {fetchThemes} from "./themeThunks";
...
+        };
+    }
+
     dispatch(loadResourceIntoTable(tableData));
 }
 
 export const loadUsersIntoTable = () => (dispatch, getState) => {
-    const {users} = getState();
+    const { users, table} = getState();
     const pagination = getTablePagination(getState());
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/thunks/tableThunks.js]: For all other resources, you changed from `getTablePagination(getState())` to `table.pagination`. Is this case different?

File: modules/admin-ui-picard/app/src/utils/resourceUtils.js [hunk 4]
```diff
@@ -98,6 +105,50 @@ export const prepareMetadataFieldsForPost = (metadataInfo, values) => {
     return metadataFields;
 }
 
+export const prepareSeriesMetadataFieldsForPost = (metadataInfo, values) => {
+    let metadataFields = [];
+
+    // fill metadataField with field information send by server previously and values provided by user
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/utils/resourceUtils.js]: grammar:  it should be `sent by server previously`, not `send by server previously`

File: modules/admin-ui-picard/app/src/utils/resourceUtils.js [hunk 5]
```diff
@@ -98,6 +105,50 @@ export const prepareMetadataFieldsForPost = (metadataInfo, values) => {
...
+        if (!!metadataInfo[i].translatable) {
+            fieldValue = {
+                ...fieldValue,
+                translatable: metadataInfo[i].translatable
+            };
+        }
+        if (!!metadataInfo[i].collection) {
+            fieldValue = {
+                ...fieldValue,
+                collection: [],
```
Comment (SimoneSchmidt) [inline modules/admin-ui-picard/app/src/utils/resourceUtils.js:132]: is this collection supposed to be an empty array?

**Review by SimoneSchmidt — CHANGES_REQUESTED** [review]

File: modules/admin-ui-picard/app/src/utils/resourceUtils.js (hunk 5 above)
Comment (isabellakutger) [reply modules/admin-ui-picard/app/src/utils/resourceUtils.js:132]: yes, it's supposed to be empty. The endpoint wants a collection field if the metadata has a collection for value options (see the call in old UI).

**Review by SimoneSchmidt — APPROVED** [review]

**Pull request merged**